ui/app/src/components/Navigation/PathNavigator/PathNavigatorItem.tsx

```diff
@@ -122,32 +124,34 @@ export default function(props: NavItemProps) {
             <MoreVertIcon className={classes.icon} />
           </IconButton>
         )}
-        <Tooltip
-          title={
-            isLeaf ? (
-              <FormattedMessage id="navigator.isLeaf" defaultMessage="Item has no children" />
-            ) : (
-              <FormattedMessage id="words.view" defaultMessage="View" />
-            )
-          }
-        >
-          <IconButton
-            aria-label="Options"
-            className={classes.itemIconButton}
-            onClick={(event) => {
-              event.stopPropagation();
-              if (isLeaf) {
-                return;
-              } else if (navigable || folder) {
-                onChangeParent?.(item);
-              } else if (previewable) {
-                onPreview(item);
-              }
-            }}
+        {showArrow && (
+          <Tooltip
+            title={
+              isLeaf ? (
+                <FormattedMessage id="navigator.isLeaf" defaultMessage="Item has no children" />
+              ) : (
+                <FormattedMessage id="words.view" defaultMessage="View" />
+              )
+            }
           >
-            <ChevronRightRoundedIcon className={classes.icon} />
-          </IconButton>
-        </Tooltip>
+            <IconButton
+              aria-label="Options"
+              className={classes.itemIconButton}
+              onClick={(event) => {
+                event.stopPropagation();
+                if (isLeaf) {
+                  return;
+                } else if (navigable || folder) {
+                  onChangeParent?.(item);
+                } else if (previewable) {
+                  onPreview(item);
```

Comment:
Why would the view children button incur in previewing?

Reply:
when it is a taxonomy, asset, script, etc